src/libsystemd/sd-device/device-enumerator.c

```diff
@@ -862,8 +862,11 @@ static int parent_crawl_children(sd_device_enumerator *enumerator, const char *p
         int r = 0;
 
         dir = opendir(path);
-        if (!dir)
-                return log_debug_errno(errno, "sd-device-enumerator: Failed to open parent directory %s: %m", path);
+        if (!dir) {
+                if (errno == ENOENT)
+                        return 0;
```

Comment:
might still be a reason to log though, even if not failing

Reply:
#24609.